### PR TITLE
Add new endpoint for delete an object verbed by actor

### PR DESCRIPTION
**Pull request by dughetti**
I created an endpoint with object, verb and actor as parameter that let us delete an object owned by actor.

### CODE REVIEW

**Comment by lsvx** [comment]
Have you tested the cache busting for this? I think this will bust the activity and not the object.

**Comment by dughetti** [comment]
I will check it  :+1:

**Comment by lsvx** [comment]
I just re-familiarized myself and yes, we need to eliminate `req.param('actor')` and `req.param('verb')` from the request object to ensure that we bust the object. Also, we don't need to pass the empty data object we can do: `Caching.bust(modifiedReq)`.

**Comment by dughetti** [comment]
haha I was doing exactly that.

File: api/controllers/ObjectController.js [hunk 1]
```diff
@@ -90,7 +89,44 @@ module.exports = {
...
+     * @param  {String} actor_id [The id of the actor of the object you want to delete]
+     * @param  {String} object [The type of object you want to delete]
+     * @param  {String} object_id [The id of the object you want to delete]
+     * @return {HTML} 200, 404, 500 [200 OK if the deletion worked, 404 object is not found, and 500 if there was an error]
+     */
+    deleteSpecificObjectVerbedByActor: function(req, res) {
+        var obj = {}, q;
 
+        q = [
+            'MATCH (a:' + req.param('actor') + ')-[v:' + req.param('verb') + ']->(o:' + req.param('object') + ')',
```
Comment (maxicecilia) [inline api/controllers/ObjectController.js]: Add verb to the docstring

**Comment by maxicecilia** [comment]
@dughetti this needs some unit testing.

**Comment by dughetti** [comment]
:+1: perfect. I will add a unit test for this.

**Pull request closed without merging**